openflow15/nxt_message.go

```diff
@@ -775,14 +773,17 @@ type PacketIn2PropPacket struct {
 }
 
 func (p *PacketIn2PropPacket) Len() (n uint16) {
-	n = p.PropHeader.Len() + p.Packet.Len()
+	return uint16(p.ActLength())
+}
 
+func (p *PacketIn2PropPacket) ActLength() int {
```

Comment:
Looking at the code I am having a hard time understanding why we need the `ActLength` method instead of just putting the correct implementation in the `Len` method for `PacketIn2PropPacket`? Then we would not need `getPacketIn2PropertyLength`?

Reply:
The type of return value for function `Len` is uint16, it always map to the value of field "Length". And the function `Len` is defined in the interface Message. If we update the value type from uint16 to int, the impact is huge, we need to update almost all messages.